### PR TITLE
Fix exercise descriptions in patternmatching.json

### PR DESCRIPTION
**Pull request by luizpericolo**
Fix some exercise descriptions making one more explicit and fixing others grammar

### CODE REVIEW

File: app/json/range.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "title": "Range",
   "modules": [
     {
-      "preparagraph": "Ranges are not inclusive at end of range:",
+      "preparagraph": "A range's upper bound are is not inclusive:",
```
Comment (dialelo) [inline app/json/range.json]: I think the "are" shouldn't be there, right?

**Comment by dialelo** [comment]
Besides the minor nitpick looks good to me, can you fix it so I merge the pull request? Thanks a lot for your time!

**Comment by luizpericolo** [comment]
Ooops, my bad!

Sure, I'll fix it!

Cheers!

Atenciosamente,
________________________________
--
Luiz Felipe da Costa Pericolo Barbosa
"Are you suggesting coconuts migrate?"
"Stay hungry. Stay foolish."

**Comment by dialelo** [comment]
Thanks a lot for your time, we're merging this in #148!

**Comment by luizpericolo** [comment]
No problem at all!
Thank you for helping us learn Scala!

Cheers!

Atenciosamente,
________________________________
--
Luiz Felipe da Costa Pericolo Barbosa
"Are you suggesting coconuts migrate?"
"Stay hungry. Stay foolish."

**Pull request closed without merging**